doc/api/stream.md

```diff
@@ -1867,10 +1867,16 @@ added: REPLACEME
 * `streams` {Stream[]|Iterable[]|AsyncIterable[]|Function[]}
 * Returns: {stream.Duplex}
 
-Combines multiple streams into a `Duplex` stream. This works by
-calling `stream.pipeline` on all the passed streams and then creating
-a `Duplex` stream which writes to the first passed stream and reads from
-the stream returned from `stream.pipeline`.
+Combines two or more streams into a `Duplex` stream that writes to the 
+first stream and reads from the last. Each provided stream is piped into
+the next, using `stream.pipeline`. If any of the streams error then all
+are destroyed, including the outer `Duplex` stream.
+
+Because `stream.pipelinify` returns a new stream that in turn can (and
+should) be piped into other streams, it enables composition. In contrast,
+when passing streams to `stream.pipeline`, typically the first stream is
+a readable stream and the last a writable stream, forming a closed 
+circuit.
```

Comment:
```suggestion
Combines two or more streams into a `Duplex` stream that writes to the
first stream and reads from the last. Each provided stream is piped into
the next, using `stream.pipeline`. If any of the streams error then all
are destroyed, including the outer `Duplex` stream.

Because `stream.pipelinify` returns a new stream that in turn can (and
should) be piped into other streams, it enables composition. In contrast,
when passing streams to `stream.pipeline`, typically the first stream is
a readable stream and the last a writable stream, forming a closed
circuit.
```